app/src/main/res/values/strings.xml

```diff
@@ -42,6 +42,10 @@
     <string name="settings_backup_storage_code_dialog_message">To enable storage backup, you need to first verify your recovery code or generate a new one.</string>
     <string name="settings_backup_storage_code_dialog_ok">Verify code</string>
 
+    <string name="settings_expert_title">Expert settings</string>
```

Comment:
Maybe call it "Advanced". I'm fine either way.

Reply:
I tended to expert to make sure this setting is for people that know what they are doing and are ready to bear the consequences instead of just more advanced settings.